README.md

```diff
@@ -1,24 +1,236 @@
-# README
+<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

- [ ] You have instructions in your readme to run tests, but I couldn't find any implemented.  Kindly add some in order to ensure your app's integrity.

- [ ] On your readme, kindly include the required `rails db:create, rails db:migrate and rails db:seed` db commands that your users will need to run in order to launch the project properly. 👍

Reply:
Removed the folder as we have not implemented any tests. `README.md` file updated according to what we have implemented.